libs/designsystem/src/lib/components/segmented-control/segmented-control.component.ts

```diff
@@ -66,13 +66,16 @@ export class SegmentedControlComponent {
     return this._selectedIndex;
   }
 
-  @Input() set selectedIndex(value: number) {
-    if (value !== this._selectedIndex) {
-      this._selectedIndex = value;
+  @Input() set selectedIndex(index: number) {
+    if (index !== this._selectedIndex) {
+      this._selectedIndex = index;
       this._value = this.items[this.selectedIndex];
+      this.selectedIndexChange.emit(this.selectedIndex);
     }
   }
 
+  @Output() selectedIndexChange = new EventEmitter<number>();
```

Comment:
Could you briefly explain why this new output is needed? And it should probably be documented in the API table of the cookbook page too.